tests/unit/test_path_info.py

```diff
@@ -89,3 +90,35 @@ def test_https_url_info_str():
 def test_path_info_as_posix(mocker, path, as_posix, osname):
     mocker.patch("os.name", osname)
     assert PathInfo(path).as_posix() == as_posix
+
+
+@pytest.mark.parametrize("cls", [WebdavURLInfo])
```

Comment:
Looks like we don't need parametrization here and below.

Reply:
Fixed